assets/js/src/form_editor/blocks/blocks.jsx

```diff
@@ -46,6 +46,7 @@ export const initBlocks = () => {
   registerBlockType(firstName.name, firstName.settings);
   registerBlockType(lastName.name, lastName.settings);
   registerBlockType(segmentSelect.name, segmentSelect.settings);
+  html.settings.supports.inserter = select('mailpoet-form-editor').isUserAdministrator();
```

Comment:
Do you think this is the best place for this line? In the middle of the block registering? It feels to me like an outliner here. Would it be better maybe in the block definition?

Reply:
It can't be done in the block definition since we need the store to be initialized. I'll move the above registering and add a comment.